{{cookiecutter.project_name}}/BUILD_LINUX.md

```diff
@@ -23,17 +23,22 @@ Note: The build process can take hours.
 Assuming that Qt 5.15.2 was installed to `/opt/qt/` (if not then adjust the command accordingly):
 
 ```sh
-mkdir {{cookiecutter.app_name}}-superbuild
-cd {{cookiecutter.app_name}}-superbuild
-cmake -DCMAKE_BUILD_TYPE:STRING=Release -DQt5_DIR:PATH=/opt/qt/5.15.2/gcc_64/lib/cmake/Qt5 -S ../{{cookiecutter.github_project}} -B .
-make -j
+cmake \
+  -DCMAKE_BUILD_TYPE:STRING=Release \
+  -DQt5_DIR:PATH=/opt/qt/5.15.2/gcc_64/lib/cmake/Qt5 \
+  -S {{cookiecutter.github_project}} \
+  -B {{cookiecutter.github_project}}-SuperBuild-Release
+
+make -j<N>
```

Comment:
It could be best as is, though my usual approach is instead
```sh
cmake --build . -j<N>
```
(Actually, I usually also throw in a `--config Release` for good measure, but I have no idea whether that is useful.)